CHANGELOG.md

```diff
@@ -130,6 +130,7 @@
 * [BUGFIX] Block-builder-scheduler: Fix data loss bug in job assignment. #11785
 * [BUGFIX] Compactor: start tracking `-compactor.max-compaction-time` after the initial compaction planning phase, to avoid rare cases where planning takes longer than `-compactor.max-compaction-time` and so actual compaction never runs for a tenant. #11834
 * [BUGFIX] Ingester: Fix issue where ingesters can exit read-only mode during idle compactions, resulting in write errors. #11890
+* [BUGFIX] Distributor: Validate the RW2 symbols field, reject invalid requests that do not have empty string as the first symbol.
```

Comment:
Note: Ingest storage record format V2 is not stable. It's currently impossible for a user to produce records of that format, hence I didn't add that part of the fix to the changelog, as it can't affect anyone yet.